root/static/scripts/common/components/FingerprintTable.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React, {useEffect, useState} from 'react';
+
+import loopParity from '../../../../utility/loopParity';
+import hydrate from '../../../../utility/hydrate';
+
+
+const FingerprintTable = ({recording}: {recording: RecordingT}) => {
+  const [tracks, setTracks] = useState([]);
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  // We ensure fetch only runs client-side since it's not in node
+  useEffect(() => {
+    fetch(
+      `//api.acoustid.org/v2/track/list_by_mbid?format=json&disabled=1&jsoncallback=?&mbid=${recording.gid}`,
```

Comment:
Breaking this up with concatenation would be nice too. (For URLs with a lot of query parameters, I usually break before `?` or `&` to maintain readability.)